src/stores/Content.jsx

```diff
@@ -63,6 +64,21 @@ export default class ContentStore {
     // }
   }
 
+  stabilityFeeMarkdown = () => {
+    return (this.getHelpItem('stability-fee-information') || {}).markdown
+  }
+
+  stabilityFeeContent = () => {
+    return this.stabilityFeeMarkdown() && compiler(this.stabilityFeeMarkdown());
+  }
+
+  showStabilityFeeAlert = () => {
```

Comment:
`showStabilityFeeAlert = () => !!this.stabilityFeeMarkdown()`
does the same as what you wrote.

that aside, following https://github.com/makerdao/scd-cdp-portal/pull/31/files#r275381192 we should have localstorage logic here, on the store rather than in the component.